code/datums/wires/robot.dm

```diff
@@ -75,10 +75,10 @@
 			else if(!R.deployed) //AI shells must always have the same laws as the AI
 				R.lawupdate = FALSE
 		if (WIRE_CAMERA) // Disable the camera.
-			if(!isnull(R.camera) && !R.scrambledcodes)
-				R.camera.status = mend
-				R.camera.toggle_cam(usr, 0)
-				R.visible_message("[R]'s camera lense focuses loudly.", "Your camera lense focuses loudly.")
+			if(!QDELETED(R.builtInCamera) && !R.scrambledcodes)
+				R.builtInCamera.status = mend
+				R.builtInCamera.toggle_cam(usr, 0)
+				R.visible_message("[R]'s camera lens focuses loudly.", "Your camera lens focuses loudly.")

```

Comment:
this spelling does not exist it is actively a misspelling no matter where you're from